### PR TITLE
Revealable text, both password masking and ellipsizing details

### PR DESCRIPTION
**Pull request by stefwalter**
This manually cherry-picks:

https://github.com/kubernetes-ui/object-describer/pull/21

### CODE REVIEW

**Comment by mvollmer** [comment]
`revealable.css` is not installed.

File: pkg/kubernetes/revealable.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    content: "●●●●";
+    position: absolute;
+    left: 0px;
+    font-weight: bold;
+    letter-spacing: 2px;
+    padding-left: 2px;
+    color: black;
+    cursor: pointer;
+    line-height: 1.5;
+    opacity: 0.5;
```
Comment (mvollmer) [inline pkg/kubernetes/revealable.css]: This makes the dot appear grayish instead of black (like this text).  Is that intended?

Comment (stefwalter) [inline pkg/kubernetes/revealable.css]: Yes, that's intentional. The dots are too big, and stand out too much if they are of identical color.

**Comment by stefwalter** [comment]
Included revealable.css in debug data.

**Pull request closed without merging**